plugins/csp-simple-objects/src/utils.hpp

```diff
@@ -0,0 +1,32 @@
+////////////////////////////////////////////////////////////////////////////////////////////////////
+//                               This file is part of CosmoScout VR                               //
+//      and may be used under the terms of the MIT license. See the LICENSE file for details.     //
+//                        Copyright: (c) 2022 German Aerospace Center (DLR)                       //
+////////////////////////////////////////////////////////////////////////////////////////////////////
+
+#ifndef CSP_SIMPLE_OBJECTS_UTILS_HPP
+#define CSP_SIMPLE_OBJECTS_UTILS_HPP
+
+#include <glm/gtc/type_ptr.hpp>
+#include <glm/gtx/quaternion.hpp>
+#include "../../../src/cs-scene/CelestialBody.hpp"
+
+namespace csp::simpleobjects { 
+
+namespace utils { 
```

Comment:
These could be merged, I guess.

```suggestion
namespace csp::simpleobjects::utils { 
```